src/Halogen/Query/Event.purs

```diff
@@ -4,22 +4,22 @@ import Prelude
 
 import Data.Foldable (traverse_)
 import Data.Maybe (Maybe)
-import FRP.Event as Event
-import Web.Event.Event as E
-import Web.Event.EventTarget as ET
+import Halogen.Subscription as HS
+import Web.Event.Event as Event
+import Web.Event.EventTarget as EventTarget
 
--- | Constructs an `Event` for a DOM event. Accepts a function that maps event
+-- | Constructs an `Emitter` for a DOM event. Accepts a function that maps event
 -- | values to a `Maybe`-wrapped action, allowing it to filter events if
 -- | necessary.
-eventListenerEventSource
+eventListener
```

Comment:
I renamed this function because `eventSource` isn't a concept any longer. Another alternative would be `eventListenerEmitter`, but I found that a bit weird to read. 

Ultimately Halogen already shadows functions from other libraries (`Halogen.subscribe` for `Halogen.Subscription.subscribe`, etc.), so this seemed like the best bet.